python_modules/dagster/dagster_tests/storage_tests/utils/partition_status_cache.py

```diff
@@ -310,6 +315,37 @@ def asset1():
         assert cached_status
         assert cached_status.serialized_materialized_partition_subset is None
 
+    def test_failure_cache_on_multi_partition_backfill(self, instance):
+        partitions_def = StaticPartitionsDefinition(["fail1", "fail2"])
+
+        @asset(partitions_def=partitions_def, backfill_policy=BackfillPolicy.single_run())
```

Comment:
Comment out of ignorance: Does this also need to be tested with a `multi_run()` policy? Could anything be different?

I know for the motivating issue they had `multi_run()`.

Reply:
yeah good point - we actually don't need any backfill policy set here, the important part is creating a run with those tags that indicate that it is a partition range (setting a backfill policy causes runs to be created with those tags by the backfill daemon or DA, but doesn't matter for this test)